# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR ORGANIZATION
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
msgid ""
msgstr ""
"Project-Id-Version: \n"
"POT-Creation-Date: 2023-04-28 16:57+0800\n"
"PO-Revision-Date: 2023-04-28 17:08+0800\n"
"Last-Translator: \n"
"Language-Team: \n"
"Language: zh_CN\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: pygettext.py 1.5\n"
"X-Generator: Poedit 3.2.2\n"

#: source\api\pdocr_api.py:4
msgid "Creating ocr object."
msgstr "正在创建OCR对象"

#: source\api\pdocr_api.py:14
msgid "导入paddleocr时错误; err code: 001"
msgstr ""

#: source\api\pdocr_api.py:200
msgid "ocr device: "
msgstr "OCR 运行设备："

#: source\api\yolox_api.py:6
msgid "Creating yolox obj. It may takes a few second."
msgstr "正在创建YOLOX对象。这可能需要一些时间。"

#: source\api\yolox_api.py:22
msgid "导入torch时错误; err code: 002"
msgstr ""

#: source\api\yolox_api.py:534
msgid "Created yolox obj."
msgstr "YOLOX对象已创建"

#: source\common\flow_state.py:99
msgid "空闲"
msgstr ""

#: source\common\flow_state.py:102
msgid "正在初始化"
msgstr ""

#: source\common\flow_state.py:104
msgid "正在准备"
msgstr ""

#: source\common\flow_state.py:106
msgid "正在进行"
msgstr ""

#: source\common\flow_state.py:108
msgid "准备结束"
msgstr ""

#: source\common\flow_state.py:110
msgid "结束"
msgstr ""

#: source\common\flow_state.py:113
msgid "移动到挑战位置"
msgstr ""

#: source\common\flow_state.py:115
msgid "挑战秘境中"
msgstr ""

#: source\common\flow_state.py:117
msgid "准备领取奖励"
msgstr ""

#: source\common\flow_state.py:119
msgid "正在寻找石化古树位置"
msgstr ""

#: source\common\flow_state.py:121
msgid "移动到石化古树位置"
msgstr ""

#: source\common\flow_state.py:123
msgid "领取奖励"
msgstr ""

#: source\common\flow_state.py:125
msgid "结束秘境"
msgstr ""

#: source\common\flow_state.py:127
msgid "在地图上传送"
msgstr ""

#: source\common\flow_state.py:129
msgid "移动到目标坐标位置"
msgstr ""

#: source\common\flow_state.py:131
msgid "移动到采集物位置"
msgstr ""

#: source\common\flow_state.py:133
msgid "拾取采集物"
msgstr ""

#: source\common\flow_state.py:135
msgid "结束自动采集"
msgstr ""

#: source\common\generic_event.py:71 source\interaction\interaction_core.py:42
msgid "恢复操作"
msgstr ""

#: source\common\generic_event.py:73
msgid "是原神窗口"
msgstr ""

#: source\common\generic_event.py:73 source\interaction\interaction_core.py:44
msgid " 秒"
msgstr ""

#: source\common\generic_event.py:73 source\interaction\interaction_core.py:44
msgid "当前窗口焦点为"
msgstr ""

#: source\common\generic_event.py:73 source\interaction\interaction_core.py:44
msgid "，操作暂停 "
msgstr ""

#: source\controller\combat_controller.py:48
msgid "有人嘎了，停止自动战斗"
msgstr ""

#: source\controller\combat_controller.py:59
msgid "停止自动战斗"
msgstr ""

#: source\controller\teyvat_move_controller.py:143
#: source\controller\teyvat_move_controller_upgrad.py:138
#: source\flow\teyvat_move_flow_upgrad.py:269
msgid "已到达目的地附近，本次导航结束。"
msgstr ""

#: source\controller\teyvat_move_controller.py:149
#: source\controller\teyvat_move_controller_upgrad.py:144
#: source\flow\teyvat_move_flow_upgrad.py:282
msgid "已到达F附近，本次导航结束。"
msgstr ""

#: source\flow\collector_flow.py:220
msgid "停止自动采集"
msgstr ""

#: source\flow\collector_flow.py:224
msgid "所有角色都已嘎掉，正在中断此次采集"
msgstr ""

#: source\flow\collector_flow.py:240 source\flow\collector_flow.py:247
#: source\flow\collector_flow.py:262
msgid "重置完成。准备进行下一次采集"
msgstr ""

#: source\flow\collector_flow.py:244
msgid "Flow timeout"
msgstr "流程超时"

#: source\flow\collector_flow.py:256
msgid "有人嘎了，正在停止此次采集"
msgstr ""

#: source\flow\collector_flow.py:338
msgid "switch Flow to: AFTER_MOVETO_COLLECTOR"
msgstr ""

#: source\flow\collector_flow.py:342
msgid "collect in"
msgstr "在采集"

#: source\flow\collector_flow.py:342
msgid "failed. reason: stuck"
msgstr "失败。原因：卡住"

#: source\flow\collector_flow.py:373
msgid "switch Flow to: BEFORE_PICKUP_COLLECTOR"
msgstr ""

#: source\flow\collector_flow.py:381
msgid "switch Flow to: IN_PICKUP_COLLECTOR"
msgstr ""

#: source\flow\collector_flow.py:391
msgid "switch Flow to: AFTER_PICKUP_COLLECTOR"
msgstr ""

#: source\flow\domain_flow.py:29
msgid "请输入秘境次数"
msgstr ""

#: source\flow\domain_flow.py:44
msgid "秘境次数："
msgstr ""

#: source\flow\domain_flow.py:47
msgid "停止自动秘境"
msgstr ""

#: source\flow\domain_flow.py:107 source\flow\domain_flow_upgrad.py:51
msgid "正在开始挑战秘境"
msgstr ""

#: source\flow\domain_flow.py:252 source\flow\domain_flow_upgrad.py:93
msgid "正在开始战斗"
msgstr ""

#: source\flow\domain_flow.py:266 source\flow\domain_flow_upgrad.py:116
msgid "正在停止战斗"
msgstr ""

#: source\flow\domain_flow.py:269 source\flow\domain_flow_upgrad.py:119
msgid "等待岩造物消失"
msgstr ""

#: source\flow\domain_flow.py:277 source\flow\domain_flow_upgrad.py:163
msgid "正在激活石化古树"
msgstr ""

#: source\flow\domain_flow.py:322
msgid "秘境结束。"
msgstr ""

#: source\flow\domain_flow.py:333 source\task\domain_task.py:97
msgid "开始下一次秘境"
msgstr ""

#: source\flow\domain_flow.py:347 source\task\domain_task.py:109
msgid "次数结束。退出秘境"
msgstr ""

#: source\flow\path_recorder_flow.py:114
msgid "position added:"
msgstr "坐标已添加："

#: source\flow\path_recorder_flow.py:147
msgid "ready to start recording"
msgstr "准备开始记录"

#: source\flow\path_recorder_flow.py:150
msgid "ready to stop recording"
msgstr "准备停止记录"

#: source\flow\path_recorder_flow.py:183
msgid "start recording"
msgstr "开始记录"

#: source\flow\teyvat_move_flow.py:115
msgid "停止自动移动"
msgstr ""

#: source\flow\teyvat_move_flow.py:156 source\funclib\generic_lib.py:72
msgid "获取传送锚点失败，正在重试"
msgstr ""

#: source\flow\teyvat_move_flow.py:274
msgid "正在落地"
msgstr ""

#: source\flow\teyvat_move_flow.py:276
msgid "结束自动行走"
msgstr ""

#: source\funclib\combat_lib.py:145
msgid ""
"Located at the map boundary, the is_chara_busy function enables fuzzy "
"recognition mode."
msgstr "处于地图边界，is_chara_busy 函数启用模糊匹配。"

#: source\funclib\combat_lib.py:189
msgid "获得当前角色编号失败"
msgstr ""

#: source\funclib\combat_lib.py:501
msgid ""
"The strategy file for the current teaming is not found in the tactic folder: "
msgstr "在tactic文件夹中没有找到当前组队的tactic文件"

#: source\funclib\combat_lib.py:553
msgid "请将配对文件中的tastic_group更名为tactic_group. 已自动识别。"
msgstr ""

#: source\funclib\combat_lib.py:572
msgid "Character Key Empty Error"
msgstr ""

#: source\funclib\scene_lib.py:22
msgid "UI界面检测失败，正在尝试按esc返回"
msgstr ""

#: source\funclib\scene_lib.py:24
msgid "将在1秒后再次尝试获取UI界面"
msgstr ""

#: source\funclib\scene_lib.py:25 source\funclib\scene_lib.py:36
#: source\funclib\scene_lib.py:39
msgid "尝试次数："
msgstr ""

#: source\funclib\scene_lib.py:30 source\funclib\scene_lib.py:32
msgid "UI界面检测失败"
msgstr ""

#: source\funclib\scene_lib.py:33
msgid "将在0.2秒后再次尝试获取UI界面"
msgstr ""

#: source\interaction\capture.py:66
msgid "Fail to get capture: "
msgstr "截图失败"

#: source\interaction\capture.py:68
msgid " waiting 2 sec."
msgstr "等待2秒"

#: source\interaction\capture.py:109
msgid "research handle: "
msgstr "重新搜索句柄"

#: source\interaction\capture.py:115
msgid "research monitor: "
msgstr ""

#: source\interaction\capture.py:123
msgid "multiple monitor detected: "
msgstr ""

#: source\interaction\capture.py:144
msgid ""
"You seem to have monitor scaling set? It is automatically recognized and "
"this does not affect usage."
msgstr "你看起来使用了多个不同的显示器缩放？已自动识别。"

#: source\interaction\interaction_core.py:44
msgid "不是原神窗口"
msgstr ""

#: source\interaction\interaction_core.py:355
msgid "appear then click fail"
msgstr "appear then click失败"

#: source\operator\pickup_operator.py:53
msgid "正在设置时间为夜晚"
msgstr ""

#: source\operator\pickup_operator.py:80
msgid "停止自动拾取"
msgstr ""

#: source\operator\pickup_operator.py:118
msgid "PICKUP_TIMEOUT_001"
msgstr ""

#: source\operator\pickup_operator.py:120
#: source\operator\pickup_operator.py:126
msgid "停止拾取"
msgstr ""

#: source\operator\pickup_operator.py:124
msgid "PICKUP_TIMEOUT_002"
msgstr ""

#: source\operator\pickup_operator.py:133
msgid "已找到物品且无法找到下一个物品，停止拾取"
msgstr ""

#: source\operator\pickup_operator.py:184
msgid "pickup: "
msgstr "拾取到："

#: source\operator\pickup_operator.py:186
msgid "已找到："
msgstr ""

#: source\task\domain_task.py:73
msgid "找不到秘境名称，放弃选择。"
msgstr ""

#: source\task\domain_task.py:128
msgid "Unknown UI page"
msgstr "未知的UI页面"

#: source\task\ley_line_outcrop\ley_line_outcrop.py:107
msgid "HandleExceptionInMission"
msgstr "在Mission中捕获到异常"

#: source\task\task_manager.py:37
msgid "No Task Running"
msgstr "没有任务正在运行"

#: source\task\task_manager.py:39
msgid "Loading"
msgstr "加载中"

#: source\task\task_manager.py:79
msgid " Start."
msgstr "启动。"

#: source\task\task_manager.py:79
msgid "Task"
msgstr "任务"

#: source\task\task_manager.py:88
msgid "End Task"
msgstr "任务结束"

#: source\webio\page_manager.py:31
msgid "Main"
msgstr "主页面"

#: source\webio\page_manager.py:32
msgid "Setting"
msgstr "设置页面"

#: source\webio\page_manager.py:33
msgid "CombatSetting"
msgstr "自动战斗设置"

#: source\webio\page_manager.py:34 source\webio\pages.py:750
msgid "CollectorSetting"
msgstr "自动采集设置"

#: source\webio\pages.py:51 source\webio\pages.py:54 source\webio\pages.py:327
msgid "未找到会话，可能由于窗口关闭。请刷新页面重试。"
msgstr ""

#: source\webio\pages.py:59
msgid "正在导入模块, 可能需要一些时间。"
msgstr ""

#: source\webio\pages.py:61
msgid "在导入完成前，请不要切换页面。"
msgstr ""

#: source\webio\pages.py:111
msgid "Get IP address"
msgstr "获取IP地址"

#: source\webio\pages.py:117
msgid "Domain Task"
msgstr "秘境任务"

#: source\webio\pages.py:121
msgid "Daily Commission"
msgstr "每日委托"

#: source\webio\pages.py:125
msgid "Claim Reward"
msgstr "领取日常奖励"

#: source\webio\pages.py:129
msgid "Ley Line Outcrop"
msgstr "地脉衍出"

#: source\webio\pages.py:133
msgid "Mission"
msgstr "Mission"

#: source\webio\pages.py:139
msgid "Task List"
msgstr "任务列表"

#: source\webio\pages.py:140
msgid "Can only be activated from the button"
msgstr "只能通过按钮启动"

#: source\webio\pages.py:146
msgid "启动/停止Task"
msgstr ""

#: source\webio\pages.py:148
msgid "## Statement"
msgstr "## 状态"

#: source\webio\pages.py:150
msgid "任务状态"
msgstr ""

#: source\webio\pages.py:152
msgid "## Mission"
msgstr ""

#: source\webio\pages.py:156
msgid "Mission Group"
msgstr "Mission组"

#: source\webio\pages.py:163
msgid "## Function"
msgstr "## 功能"

#: source\webio\pages.py:164
msgid "Can only be activated from the hotkey '['"
msgstr "只能通过快捷键启动（默认为‘['）"

#: source\webio\pages.py:166
msgid "FlowMode"
msgstr "流程模式"

#: source\webio\pages.py:168
msgid "Idle"
msgstr "空闲"

#: source\webio\pages.py:169
msgid "AutoCombat"
msgstr "自动战斗"

#: source\webio\pages.py:186
msgid "## Log"
msgstr "## 日志"

#: source\webio\pages.py:193
msgid "更新提示"
msgstr ""

#: source\webio\pages.py:228
msgid "Not Enabled"
msgstr "未启用"

#: source\webio\pages.py:229
msgid "LAN IP"
msgstr "局域网IP"

#: source\webio\pages.py:229
msgid "WAN IP"
msgstr "以太网IP"

#: source\webio\pages.py:276
msgid "# Config"
msgstr "# 设置"

#: source\webio\pages.py:282 source\webio\pages.py:631
#: source\webio\pages.py:711 source\webio\pages.py:756
msgid "## config:"
msgstr "## 设置："

#: source\webio\pages.py:296
msgid "Combat.json"
msgstr "自动战斗设置"

#: source\webio\pages.py:297
msgid "Domain.json"
msgstr "自动秘境设置"

#: source\webio\pages.py:298
msgid "General.json"
msgstr "通用设置"

#: source\webio\pages.py:299
msgid "Keymap.json"
msgstr "快捷键设置"

#: source\webio\pages.py:300
msgid "collected.json"
msgstr "自动采集-已采集id"

#: source\webio\pages.py:301
msgid "collection_blacklist.json"
msgstr "自动采集-黑名单id"

#: source\webio\pages.py:302
msgid "collection_log.json"
msgstr "自动采集记录"

#: source\webio\pages.py:303
msgid "Collector.json"
msgstr "自动采集设置"

#: source\webio\pages.py:348 source\webio\pages.py:872
msgid "Verified!"
msgstr "验证通过！"

#: source\webio\pages.py:357 source\webio\pages.py:881
msgid "Waiting..."
msgstr "等待中..."

#: source\webio\pages.py:362 source\webio\pages.py:886
msgid "You may want to enter: "
msgstr "你可能想输入："

#: source\webio\pages.py:367 source\webio\pages.py:891
msgid "Not a valid name"
msgstr "不是一个有效的名称"

#: source\webio\pages.py:410
msgid "saved!"
msgstr "保存成功！"

#: source\webio\pages.py:453
msgid "Do you need to save changes?"
msgstr "你要保存设置吗？"

#: source\webio\pages.py:454
msgid "No"
msgstr "否"

#: source\webio\pages.py:454
msgid "Yes"
msgstr "是"

#: source\webio\pages.py:625
msgid "# Setting"
msgstr "# 设置"

#: source\webio\pages.py:695
msgid "# CombatSetting"
msgstr "# 自动战斗设置"

#: source\webio\pages.py:701
msgid "# Add team"
msgstr "# 增加队伍"

#: source\webio\pages.py:705
msgid "Add team"
msgstr "增加队伍"

#: source\webio\pages.py:707
msgid "自动填充"
msgstr ""

#: source\webio\pages.py:817
msgid "Add to blacklist"
msgstr "添加到黑名单"

#: source\webio\pages.py:825
msgid "展开/收起"
msgstr ""

#: source\webio\pages.py:860
msgid "clean list"
msgstr "清空列表"

#: source\webio\webio.py:34
msgid "webio启动完成"
msgstr ""

#~ msgid "AutoDomain"
#~ msgstr "自动秘境"

#~ msgid "AutoCollector"
#~ msgstr "自动采集"

#~ msgid "auto_aim.json"
#~ msgstr "自动瞄准设置"

#~ msgid "auto_pickup.json"
#~ msgstr "自动采集设置"

#~ msgid "auto_pickup_default_blacklist.json"
#~ msgstr "自动采集黑名单设置"

#~ msgid "config.json"
#~ msgstr "通用设置"

#~ msgid "character.json"
#~ msgstr "默认角色配置"

#~ msgid "character_dist.json"
#~ msgstr "角色名列表"

#~ msgid "team_example_2.json"
#~ msgstr "team示例文件2"

#~ msgid "Add team with characters"
#~ msgstr "增加队伍（使用character.json的配置）"

#~ msgid "Automatic generate a list of collected items"
#~ msgstr "从采集日志中自动生成已采集列表id"

#~ msgid "Automatic generate a list of blacklist items"
#~ msgstr "从采集日志中自动生成黑名单列表id"

#~ msgid "Add to collected"
#~ msgstr "添加到已采集"

#~ msgid "cleaning cvautotrack files"
#~ msgstr "正在清理cvautotrack文件"

#~ msgid "autoTrack.log 1 cleaned"
#~ msgstr "autoTrack.log 1 已清理"

#~ msgid "autoTrack.log 2 cleaned"
#~ msgstr "autoTrack.log 2 已清理"

#~ msgid "autoTrack.log 3 cleaned"
#~ msgstr "autoTrack.log 3 已清理"

#~ msgid "cvAutoTrack DLL has been loaded."
#~ msgstr "cvAutoTrack DLL 已经加载"

#~ msgid "day(s)"
#~ msgstr "天"

#~ msgid "success rate"
#~ msgstr "成功率"

#~ msgid "total num"
#~ msgstr "总次数"

#~ msgid "success num"
#~ msgstr "成功次数"

#~ msgid "fail num"
#~ msgstr "失败次数"

#~ msgid "Within"
#~ msgstr "在"

#~ msgid "Program stop."
#~ msgstr "程序终止"
